server/bundle.go

```diff
@@ -29,6 +39,14 @@ func (s *RESTServer) BundleListHandler(w http.ResponseWriter, r *http.Request, p
 // BundleListPrefixHandler handles GET requests to "/bundle/list/:prefix".
 func (s *RESTServer) BundleListPrefixHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	prefix := ps.ByName("prefix")
+
+	if s.TapeUsed == false {
+		w.WriteHeader(503)
+		fmt.Fprintln(w, items.ErrNoStore)
```

Comment:
Don't use `items` just for that error message. Use a literal string "Tape access is temporarily disabled". If you hate repeating the literal everywhere, make a function or local constant to do it.